### ISSUE TITLE
Needs ability to exclude directories

### ISSUE THREAD

**Issue by jpterry** [Invoca/magic_frozen_string_literal#2]
Many rubyists will install gems into a project local subdirectory like `vendor/bundle` with bundler. This is very useful when users don't have permissions to muck with system installed gems. Unfortunately this gem considers `vendor/bundle` as part of the application repo and attempts to add the comment to all gem files.

A solution could be to always ignore `vendor/bundle`, but a more general solution would be to allow an `--exclude` to be passed from the CLI.

A workaround is to, 1) `rm -rf vendor/bundle` 2) `magic_frozen_string_literal` 3) reinstall the local gems.

Feel free to categorize as wontfix. The code is simple enough that it might not be worth adding many features.

**Comment by ColinDKelley** [comment]
Good point. That is how our main web/ repo is set up. We hadn't noticed that because we haven't unleashed this tool there yet!
Also this tool has a stub for the exclude feature but it isn't implemented yet. Maybe it should be implemented and default to `vendor/`?
